### ISSUE TITLE
where to find the code to plot the following figure?

### ISSUE THREAD

**Issue by liamxg** [broadinstitute/pyro-cov#49]
<img width="563" alt="image" src="https://user-images.githubusercontent.com/46078579/196388981-fd8698dd-b555-46b4-9945-2ba88e306b93.png">

**Comment by fritzo** [comment]
https://github.com/broadinstitute/pyro-cov/blob/master/notebooks/mutrans.ipynb

**Comment by fritzo** [comment]
See `plot_emergence()` in that file. We do not store plots with code; rather plots are stored in [paper/](https://github.com/broadinstitute/pyro-cov/tree/master/paper) and code is stored with notebooks.

**Comment by fritzo** [comment]
yes, I believe `plot_emergence()` creates that figure.  Can you be more specific about what you're looking for and what you haven't found in `plot_emergence()`?

**Comment by Jialu-Zuo** [comment]
Hi @barkasn , @martinjankowiak , @JacobLemieux , @bkotzen , @corneliusroemer. @fritzo ,
I'm trying to plot strain emergence figure,   it seems that there is something wrong with the 95% confidence intervals. Looking forward to your explaination!

**Comment by Jialu-Zuo** [comment]
Hi @barkasn , @martinjankowiak , @JacobLemieux , @bkotzen , @corneliusroemer. @fritzo ,
I'm trying to plot strain emergence figure,   it seems that there is something wrong with the 95% confidence intervals. Looking forward to your explaination! 
<img width="461" alt="屏幕截图 2023-12-18 194817" src="https://github.com/broadinstitute/pyro-cov/assets/149792876/83a1aa4b-4dd3-436b-be1e-d250db7b374f">

**Comment by liamxg** [comment]
Dear @Jialu-Zuo,
what kind of problems?

**Comment by Jialu-Zuo** [comment]
> Dear @Jialu-Zuo, what kind of problems?
As the figure shows，the confidence intervals are really wide，I guess it is because of the small amount of sample. Besides，the "cases/day" of top ten lineages are all zeros.
![Screenshot_20231219_081335_cn edu seu iportal](https://github.com/broadinstitute/pyro-cov/assets/149792876/e8cb67b3-ed2a-4493-8692-1be622fd298b)

**Comment by Jialu-Zuo** [comment]
> Dear @Jialu-Zuo, what kind of problems?

As the figure shows，the confidence intervals are really wide，I guess it is because of the small amount of sample. Besides，the "cases/day" of top ten lineages are all zeros.
![Screenshot_20231219_081335_cn edu seu iportal_edit_236038216837941](https://github.com/broadinstitute/pyro-cov/assets/149792876/6d040df8-e886-4bde-bef1-2e16bfc23109)

**Comment by Jialu-Zuo** [comment]
Hi @barkasn , @martinjankowiak , @JacobLemieux , @bkotzen , @corneliusroemer. @fritzo ,
as the figure shows，the confidence intervals are really wide. Besides，the "cases/day" of top ten lineages are all zeros，I think it is because of the lack of the data. JHU has stopped collecting data as of 2023/3/10,  I'm also wondering if there is other appropriate data to take place of the origin. Looking forward to your reply!
<img width="441" alt="image" src="https://github.com/broadinstitute/pyro-cov/assets/149792876/ffdbb5f2-0ed3-4da6-8dc0-c26f651a8b39">
